easybuild/easyconfigs/b/Boost/Boost-1.63.0-foss-2018a-Python-2.7.14.eb

```diff
@@ -0,0 +1,23 @@
+name = 'Boost'
+version = '1.63.0'
```

Comment:
@Serpens This easyconfig file can be removed now?

Reply:
Yes